### PR TITLE
Button component

### PR DESCRIPTION
**Pull request by jcasado6**
changed around class names and reorganized css per your messages yesterday.

### CODE REVIEW

**Comment by 0xZakk** [comment]
@jcasado6 Is this a new PR? It looks like there aren't any changes requested here

**Review by 0xZakk — CHANGES_REQUESTED** [review]
See my comment

**Comment by jcasado6** [comment]
Yes, this is a new PR with changes because the original one was closed due to the accidental merge.

**Review by mrrogercampbell — CHANGES_REQUESTED** [review]
Hey @jcasado6 it doesnt look like any of your css is being utilized. All I see is a white button.

**Review by mrrogercampbell — CHANGES_REQUESTED** [review]
Hey @ZakkMan I just re-reviewed @jcasado6 code and it looks good. 

Only thing is with him recreating this PR none of his commit history is showing. Not sure why. 

I feel like this is ready to be merged in to develop.

**Review by mrrogercampbell — APPROVED** [review]
@ZakkMan I believe this component is complete.

**Review by 0xZakk — CHANGES_REQUESTED** [review]
@jcasado6 Overall this looks really good. The font looks a little off though. Can you verify that this is the correct font?

**Comment by jcasado6** [comment]
updated large button font family

File: client/src/components/Button/Button.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  font-family: Raleway;
+}
+
+.button-light-pink {
+  background-color: #fae5dc;
+  color: #441f0e;
+  font-weight: bold;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  height: 31px;
+  width: 88px;
```
Comment (0xZakk) [inline client/src/components/Button/Button.css]: Can you set padding here rather than a hard height and width?

Comment (0xZakk) [reply client/src/components/Button/Button.css]: This is for all of the buttons where you have an absolute value for width and height

File: client/src/components/Button/Button.stories.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import Button from "./Button";
+import "./Button.css";
+
+storiesOf("Button", module)
+  .add("light-pink", () => <Button label="Book Now" type="light-pink" />)
```
Comment (0xZakk) [inline client/src/components/Button/Button.stories.js]: Can you make these labels more human-friendly? Like `Light Pink`, `Dark Pink`, `Red`, etc

**Review by 0xZakk — CHANGES_REQUESTED** [review]
@jcasado6 Just a few small things

**Comment by jcasado6** [comment]
changed height&width to padding, changed story labels.

**Review by 0xZakk — APPROVED** [review]

**Pull request merged**